SPDLOG_QUALITY_DECLARATION.md

```diff
@@ -29,16 +29,15 @@ For the compiled portion of `spdlog`, the `SPDLOG_API` C++ macro indicates the s
 
 ### API Stability Policy [1.iv]
 
-There is no published policy for API stability in `spdlog`.
+There is no policy for API stability. This is not a problem because the `spdlog_vendor` package importing the `spdlog dependency` is using a fixed version, in this case, the [1.5.0](https://github.com/gabime/spdlog/releases/tag/v1.5.0).
```

Comment:
I think there is no problem with pinning a version, but IMO the wording should reflect that it won't affect ROS 2 users of `spdlog_vendor`. @brawner may suggest a better wording.

Reply:
I used the same wording that you can find in [libyaml](https://github.com/ros2/libyaml_vendor/blob/master/libyaml_q_declaration.md#api-stability-policy-1iv)

Reply:
```suggestion
There is no policy for API stability. However, the vendored package `spdlog_vendor` package importing the `spdlog dependency` is using a fixed version, in this case, [1.5.0](https://github.com/gabime/spdlog/releases/tag/v1.5.0), which will provide API stability to downstream packages.
```